app/src/main/java/com/ls/ui/fragment/LocationFragment.java

```diff
@@ -110,23 +118,36 @@ private void fillMapViews(List<Location> locations) {
         uiSettings.setMyLocationButtonEnabled(false);
         uiSettings.setCompassEnabled(false);
         uiSettings.setRotateGesturesEnabled(false);
+
+        mGoogleMap.setOnMarkerClickListener(new GoogleMap.OnMarkerClickListener() {
+
+            @Override
+            public boolean onMarkerClick(Marker marker) {
```

Comment:
Here also add `marker.showInfoWindow();`  to show info window of marker because currently it is not showing on which marker user have clicked.